### PR TITLE
Fix cluster sharding benchmark

### PR DESCRIPTION
**Pull request by Arkatufus**
## Changes

Fix broken cluster sharding benchmark

### CODE REVIEW

File: src/benchmark/Akka.Benchmarks/Configurations/Configs.cs [hunk 1]
```diff
@@ -37,16 +37,19 @@ public string GetValue(Summary summary, BenchmarkCase benchmarkCase, SummaryStyl
...
-                var operationsPerSecond = 1 / (nsPerOperation / 1e9);
-
-                return operationsPerSecond.ToString("N2");  // or format as you like
-            }
+            if (!summary.HasReport(benchmarkCase)) 
+                return "<not found>";
+            
+            var report = summary[benchmarkCase];
+            var statistics = report?.ResultStatistics;
+            if(statistics is null)
```
Comment (Arkatufus) [inline src/benchmark/Akka.Benchmarks/Configurations/Configs.cs:45]: Both report and statistics can be null if a benchmark case failed to run.

File: src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardingInfrastructure.cs [hunk 2]
```diff
@@ -33,11 +33,11 @@ public ResolveResp(string entityId, Address addr)
             public Address Addr { get; }
         }
         
-        public ShardedEntityActor()
+        public ShardedEntityActor(string entityId)
         {
-            Receive<ShardingEnvelope>(e =>
+            Receive<Resolve>(_ =>
```
Comment (Arkatufus) [inline src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardingInfrastructure.cs:38]: This is the cause for the bug.

`ShardingEnvelope` does not get passed to the shard entity anymore, it is automatically unwrapped by the new extractor.

**Review by Arkatufus — COMMENTED** [review]
Self review

File: src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardingInfrastructure.cs (hunk 2 above)
Comment (Aaronontheweb) [reply src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardingInfrastructure.cs:38]: BITTEN BY MY OWN BUG FIX

Comment (Aaronontheweb) [reply src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardingInfrastructure.cs:38]: Thank you @Arkatufus

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**